docs/releases.md

```diff
@@ -51,9 +53,23 @@ At this stage, we consider the release to be "production ready" and ask will ask
 
 At this stage, the release is "battle hardened" and ready for wide deployment.
 
+## Release Cycle
```

Comment:
Do you want to say something here about patch fixes and how long we'll continue to patch a release? Maybe until the next release lands (aka we'll potentially ship patches for ~6 weeks after launch)?